frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -200,13 +201,23 @@ public void pollBatch(String threadId, long waitTime, Map<String, Job> jobsRepo)
             logger.trace("get first job: {}", Objects.requireNonNull(j).getJobId());
 
             jobsRepo.put(j.getJobId(), j);
+            // describe request job batch size always is 1
+            if (j.getCmd() == WorkerCommands.DESCRIBE) {
+                return;
+            }
             long begin = System.currentTimeMillis();
             for (int i = 0; i < batchSize - 1; ++i) {
                 j = jobsQueue.poll(maxDelay, TimeUnit.MILLISECONDS);
                 if (j == null) {
                     break;
                 }
                 long end = System.currentTimeMillis();
+                // describe request job batch size always is 1
+                if (j.getCmd() == WorkerCommands.DESCRIBE) {
+                    // Add the job back into the jobsQueue
+                    jobsQueue.addFirst(j);
+                    break;
+                }
```

Comment:
Seems like dead code to me given we are returning for `cmd == DESCRIBE` above

Reply:
The batch size of a jobsRepo containing "WorkerCommands.DESCRIBE" should always be 1. This is the goal of this code section.

There are two scenarios.
1. the first incoming request in jobsQueue  is WorkerCommands.DESCRIBE. line204-207 covers this case.
2. "WorkerCommands.DESCRIBE" is in the middle of jobsQueue. line215-220 covers this case. In this case, we need stop add request into jobsRepo, and put "WorkerCommands.DESCRIBE" back to the first of jobsQueue.